civihr_default_theme/scss/components/_leave-request-modal.scss

```diff
@@ -16,6 +16,10 @@ $row-margin-bottom: 15px;
   margin-bottom: 20px;
 }
 
+.chr_leave-request-modal__footer {
+  margin: 0;
```

Comment:
why was the margin off in the first place? What caused it? This should be a standard bootstrap modal, the style should work as-is